utility/service/service.go

```diff
@@ -0,0 +1,148 @@
+package service
+
+import (
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/utility/types"
+)
+
+type Relay interface {
+	RelayPayload
+	RelayMeta
+}
+
+type RelayPayload interface {
+	GetData() string               // the actual data string for the external chain
+	GetMethod() string             // the http CRUD method
+	GetPath() string               // the REST Path
```

Comment:
I think we should say `HTTP Path` unless it is actually rest.

Source:  https://htmx.org/essays/how-did-rest-come-to-mean-the-opposite-of-rest/?utm_source=pocket_mylist

<img width="646" alt="Screen Shot 2022-11-02 at 7 59 28 PM" src="https://user-images.githubusercontent.com/1892194/199639727-8930c114-897b-4a64-82cd-06d091cd00ae.png">